ITEST/itest.storelogdata/pom.xml

```diff
@@ -58,24 +56,6 @@
     </build>
 
     <dependencies>
-        <dependency>
-            <groupId>org.testcontainers</groupId>
-            <artifactId>kafka</artifactId>
-            <version>${testcontainers.version}</version>
```

Comment:
Why it was moved to the parent module? Is it required for all tests, including WebStudio?